### PR TITLE
Add alanrhannah-edinburgh file

### LINKED ISSUE #5: alanrhannah's workflow adding Edinburgh to the repository
## Edinburgh
- [x] Create an issue
- [x] Create a branch
- [x] Make changes
- [x] open a pr
- [ ] collaberate
- [ ] get approval to ship :ship: (:+1:) 
- [ ] merge

### PR DESCRIPTION
**Pull request by alanrhannah**
closes #5

### CODE REVIEW

File: alanrhannah-edinburgh.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+## Edinburgh
+
+### Restaurants
+[The Kitchen](https://thekitchin.com/)
```
Comment (alanrhannah) [inline alanrhannah-edinburgh.md:4]: New lines!

File: alanrhannah-edinburgh.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+[Castle Terrace](https://castleterracerestaurant.com/)
+
+
+### Places to go
+Edinburgh Castle
```
Comment (alanrhannah) [inline alanrhannah-edinburgh.md]: make this an `<ul>`

**Review by alanrhannah — COMMENTED** [review]
Edinburgh isn't as hot

**Review by alanrhannah — COMMENTED** [review]
All good.

**Review by smurffy — APPROVED** [review]
LGTM

**Pull request merged**